index.html

```diff
@@ -0,0 +1,262 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/js/bootstrap.bundle.min.js"></script>
+    <link rel="stylesheet" href="styles.css" />
+    <title>Web Development Course</title>
+  </head>
+  <body>
+    <div class="main-container">
+
+      <div class="side-nav open">
+        <ul class="side-nav-items-container">
+          <li class="side-nav-close-btn-container side-nav-size">
+            <img src="images/close_mobile_menu.png" alt="close button" class="side-nav-close-btn">
+          </li>
+          <li class="side-nav-about-container side-nav-size">
+            <a href="about.html" class="sid-nav-about">About</a>
+          </li>
+          <li class="side-nav-program-container side-nav-size">
+            <a href="#" class="sid-nav-program">Program</a>
+          </li>
+          <li class="side-nav-join-container side-nav-size">
+            <a href="#" class="sid-nav-join">Join</a>
```

Comment:
As the l[ast code reviewer suggested](https://github.com/EngruugbeJimmy/Opolo_Conference_Webpage/pull/1#discussion_r1007973595) kindly remove the withe spacing on the right side of your project. :eyes: :eyes: 
![image](https://user-images.githubusercontent.com/98363075/198755252-9b7c0773-4d6c-4800-899c-08594acef296.png)